pkg/dag/dag_uki_boot.go

```diff
@@ -16,7 +16,7 @@ func RegisterUKI(s *state.State, g *herd.Graph) error {
 	s.LogIfError(s.UKIMountBaseSystem(g), "mounting base mounts")
 
 	// Move to sysroot
-	s.LogIfError(s.UkiPivotToSysroot(g), "pivot to sysroot")
+	s.LogIfError(s.UKIPivotToSysroot(g), "pivot to sysroot")
```

Comment:
changed it because all others were named UKI..., it's also a more Golang approach to capitalize these for what I understand